dvc_ssh/__init__.py

```diff
@@ -108,11 +87,6 @@ def _prepare_credentials(self, **config):
 
         login_info["gss_auth"] = config.get("gss_auth", False)
         login_info["agent_forwarding"] = config.get("agent_forwarding", True)
-        login_info["proxy_command"] = user_ssh_config.get("ProxyCommand")
```

Comment:
We don't have a config for this in dvc. `ProxyCommand` is supported in asyncssh AFAIK.

Reply:
I think this was copied straight from `paramiko` based filesystem. 
I see that asyncssh got `proxy_command` support as well as OpenSSH's `ProxyCommand` support at the same time: ronf/asyncssh#376 (commit https://github.com/ronf/asyncssh/commit/47375c40786996007cd487b3937a2a1eb785f30d).